community/gardening/slack-map.md

```diff
@@ -1,28 +1,29 @@
 ---
 layout: single
-title:  "Master Schedule for Spinnaker Gardening Days #communityHack"
+title:  "Slack Channels @ Spinnaker Gardening Days"
 sidebar:
   nav: community
 ---
 ## Slack Map
-Your guide to Gardening Slack channels in [Spinnaker Slack](https://join.spinnaker.io)
+Your guide to all the Gardening Slack channels in [Spinnaker Slack](https://join.spinnaker.io)
+
 
 |Slack Channel|Description|
 |-----|-----------------------|
-|#gardening-ineedhelp|Event helpdesk|
-|#gardening-team-[projectString]|Team channels|
-|#gardening-idea-[projectString]|Brainstorming channels|
-|#gardening-general|General event chat|
-|#gardening-spinnaker-tips|Share Spinnaker tips for Spinnaker.io & Twitter mention |
-|#gardening-training|Training questions and discussion|
-|#gardening-stackoverflow|Share stackoverflow links for answer credit|
-|#gardening-LOLs|Share jokes and media to make each other laugh|
-|#gardening-gamers|Gamer chat|
-|#gardening-rave|Dance party|
-|#gardening-karaoke|Karaoke!|
-|#gardening-yoga-mindfulness|Scheduled yoga & mindfulness breaks|
-|#gardening-anonymous|Anonymous feedback|
-|#spinnaker-news|Share Spinnaker news & blog content for April newsletter|
-|#sig-documentation|Documentation SIG-hosted activities and chatter|
-|#sig-kubernetes|Kubernetes SIG-hosted chatter|
-|#sig-ui|UI SIG-hosted chatter|
+|[#gardening-ineedhelp](slack://channel?team={spinnakerteam}&id={gardening-ineedhelp})|Event helpdesk|
```

Comment:
These links don't work for me. They open Slack but don't switch channels or workspaces. I recommend just grabbing the channel URL's directly from Slack and using those by right clicking on a channel name in Slack and copying the channel link. You'll get the URL for a channel like gardening-ineedhelp, https://spinnakerteam.slack.com/archives/CURFZGL2E

Reply:
updated. thanks for this suggestion :woman-facepalming::skin-tone-3: